packages/core/src/App/Components/Layout/Header/wallets/account-actions-wallets.tsx

```diff
@@ -0,0 +1,94 @@
+import React from 'react';
+import { Button, Icon, Popover } from '@deriv/components';
+import { routes } from '@deriv/shared';
+import { localize, Localize } from '@deriv/translations';
+import { observer, useStore } from '@deriv/stores';
+import { LoginButton } from '../login-button.jsx';
+import { SignupButton } from '../signup-button.jsx';
+import { BinaryLink } from '../../../Routes/index.js';
+import ToggleNotifications from '../toggle-notifications.jsx';
+import AccountInfoWallets from './account-info-wallets';
+import 'Sass/app/_common/components/account-switcher.scss';
+
+const AccountActionsWallets = observer(() => {
+    const { client, ui, notifications } = useStore();
+    const { is_logged_in, accounts, loginid } = client;
+    const { openRealAccountSignup, toggleAccountsDialog, is_mobile, is_accounts_switcher_on } = ui;
+    const { is_notifications_visible, notifications: notificationsArray, toggleNotificationsModal } = notifications;
+
+    const notifications_count = notificationsArray?.length;
+
+    const active_account = accounts?.[loginid ?? ''];
+    const is_virtual = active_account?.is_virtual;
+    const currency = active_account?.currency;
+
+    if (is_logged_in) {
+        return is_mobile ? (
+            <React.Fragment>
+                <ToggleNotifications
+                    count={notifications_count}
+                    is_visible={is_notifications_visible}
+                    toggleDialog={toggleNotificationsModal}
+                    tooltip_message={undefined}
+                />
+                <React.Suspense fallback={<div />}>
```

Comment:
future improvement / next pr: consider a loader?

Reply:
@wojciech-deriv OMG, forgot to delete it. We don't need this anymore because I've added loader inside of AccountInfoWallets component. Updated PR